src/context/tasks.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from "react";
+
+export const TodoContext = React.createContext({});
+
+
+const TodoProvider: React.FC<React.ReactNode> = ({ children }) => {
+  const [todos, setTodos] = React.useState<any>({
+    id: Math.random(), 
+    title: "abc",
+    description: "xyz",
+    status: false
+  });
+
+  
+   const setTasks = (tasksContext: any = []) => {
```

Comment:
Dont use 'any' all the time if you are aware about the incoming data or value then properly define the structure. Try to avoid using any as much as possible
You can have separate model file where you can define your interfaces